test/typescript/edge/main.ts

```diff
@@ -99,6 +99,17 @@ export class ReferenceStack extends TerraformStack {
       req: set.set,
       singlereq: { reqbool: true, reqnum: 1, reqstr: "reqstr" },
     });
+
+    // passing a list ref of a complex list type (no block) into an output
+    new TerraformOutput(this, "list_from_list_type_ref", {
```

Comment:
Why not have this case in the other language tests?

Reply:
Ah, good idea